### DISCUSSION TITLE
#multiple git repos and submodule support

### DISCUSSION THREAD

**Discussion by imabusyman** [microsoft/aspire#4147] [Ideas]
I know this might be a repeat somewhere, but the ability to have git repos support.  Our microservices are in their own git repo and the idea to bring them into one git repo isn't ideal at all.  We have a lot of microservices we want to put in aspire (if makes sense) and containerize them, but still leave them in their own repo due to owning by different teams....

Submodule support as well, not sure if this in the plans, but one of our major blockers to POC out aspire.

**Comment by davidfowl** [comment]
Join the discussion here https://github.com/dotnet/aspire/discussions/1137. Some people have built prototypes that automatically clone repositories
